vite.config.ts

```diff
@@ -44,4 +44,14 @@ export default defineConfig({
       },
     ],
   },
+  server: {
```

Comment:
I don't think you can do this too since server is static build ( you can ask freeman to proxied this)